operator/src/test/java/oracle/kubernetes/operator/helpers/JobHelperTest.java

```diff
@@ -1045,6 +1048,87 @@ void whenIntrospectRequestSet_runIntrospector() {
     assertThat(job, notNullValue());
   }
 
+  @Test
+  void whenDomainIsIstioEnabled_localhostBindingsDisable_hasIstioUseLocalhostBindingsEnv() {
+    configureDomain()
```

Comment:
There is no need to break this on to a second line; that only makes sense when there are multiple calls that make it impossible to be a single line.

Reply:
Fixed to be on single line.